cs/API-v2/Program.cs

```diff
@@ -0,0 +1,155 @@
+﻿using System;
+using System.IO;
+using System.Net.Http;
+using System.Security.Cryptography;
+using System.Text;
+using System.Collections.Generic;
+using System.Linq;
+using Org.BouncyCastle.Crypto;
+using Org.BouncyCastle.Crypto.Parameters;
+using Org.BouncyCastle.OpenSsl;
+using Org.BouncyCastle.Security;
+
+namespace WalutomatApiV2Example
+{
+    class Program
+    {
+        static string host = "https://api.walutomat.pl";
+        static void Main(string[] args)
+        {
+            switch (args[0])

```

Comment:
`switch (args.Length > 0 ? args[0] : "")` or it throws when no args were provided